frontend/src/tests/import-export-data/existing-objects-for-import.js

```diff
@@ -1,17 +1,15 @@
 /*
- * A collection of context (existing objects)
+ * A collection of existing objects
  * used for computing difference when importing objects.
  */
 
-export const importInstructorContext = {
+export const existingObjectsForInstructorImport = {
```

Comment:
I'm still unclear on this. So we have some data, and we have some modified data. We use the `diff` routine on both and expect to get something specific out. Right?

If so, we should co-locate this data. We can have an object with keys `data1` and `data2` and `expectedDiff`. Then, when one of the tests fail, someone will actually know where to look.

Reply:
do you mean instead of taking snapshots on the computed diff, I should take snapshots of a constructed object, which is:
```
{
existingData: existingInstructors,
modifedData: normalizedInstructors,
expectedDiff: diffImport(normalizedInstructors, existingInstructors)
}
```?

Reply:
Snapshots of the diffs are okay, but you should at lease co-locate the changed and unchanged data. Note `existingData` is a bad name. It doesn't exist until you define it, but the word `existing` makes it sound like it comes from somewhere. `initialData` and `modifiedData` would be better names.